Dockerfile

```diff
@@ -38,8 +38,11 @@ WORKDIR /go/src/github.com/vmware-tanzu/velero
 
 COPY . /go/src/github.com/vmware-tanzu/velero
 
-RUN mkdir -p /output/usr/bin && \
-    export GOARM=$( echo "${GOARM}" | cut -c2-) && \
+# if TARGETVARIANT is v8, set GOARM to empty string
+RUN if [ "${TARGETVARIANT}" = "v8" ]; then \
+        export GOARM=""; \
```

Comment:
Should here have an `else export GOARM=$( echo "${GOARM}" | cut -c2-)`? (what it was doing prior to the change)

Reply:
Adding it back now :) will getting a commit up with the changes soon!

Reply:
added